packages/vote/net/query.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package net;
+
+option go_package = ".;net";
+
+service VoterQuery {
+    rpc Opinion (QueryRequest) returns (QueryReply) {}
+}
+
+message QueryRequest {
+    repeated string id = 1;
+}
+
+message QueryReply {
+    repeated int32 opinion = 1;

```

Comment:
That's unfortunate. Doesn't protobuf allow to use something smaller than 4 bytes?

Reply:
Unfortunately not, however, over the wire, protobuf truncates the amount of bytes used to the amount required. So this would actually be one byte.